packages/types/src/codec/createType.ts

```diff
@@ -262,6 +262,11 @@ export function createClass (type: Text | string): Constructor {
   );
 }
 
+// alias for createClass
+export function ClassOf (name: string): Constructor {
```

Comment:
Unused atm. However just an alias so you could do `MyCoolType: ClassOf('AttenstedCandidate')` - this is actually sorely needed, i.e. in the case of `Balance` we are currently stuck at `u128` with no real way to override this from an API-user POV.

Reply:
Agreed, this is cool